Automattic/pocket-casts-android
# Widget -  empty state color update

## Description

This 
- Adds dynamic color to the empty state widget app icon image
- Reduces play button size 

## Testing Instructions
1. Install the app 
2. Do not select any episode to play
3. Long press the app icon 
4. Tap Widgets
5. Drag the widget to the home screen
6. ✅ Notice that the widget empty state app icon has the dynamic color 
7. Play an episode in the app
8. ✅ Notice that the play icon is smaller than before (https://github.com/Automattic/pocket-casts-android/pull/1588)


## Screenshots or Screencast 
<img width = 300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/e87c175a-1629-4cc0-9d2c-6482e0b09e4b"/>
<img width = 300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/73adc5ea-1d15-419b-b605-43806a80e518"/>

## Checklist
 
#### I have tested any UI changes...
